src/operations/sepia_tone.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{arg_parsers::SepiaThreshold, error::MagickError, image::Image};
+
+pub fn sepia_tone(image: &mut Image, _threshold: &SepiaThreshold) -> Result<(), MagickError> {
```

Comment:
For what it is worth, I found sepia implementations in
https://github.com/mimanshu-maheshwari/plutofilter-rs/blob/06c4bdd5e341282295d3e6802e14811d9d2e6d7a/src/surface.rs#L413
and https://github.com/silvia-odwyer/photon/blob/b4e3b29c2006b224a6c1e16bc6bbe891ae8698fb/crate/src/monochrome.rs#L83
(particularly ignoring the IM implementation once again).

Could go with something of either of the two, yet only the first one seems to have something like a cut-off value maybe similar to the IM threshold?